### PR TITLE
Fix flaky TTS test

### PR DESCRIPTION
**Pull request by Hamms**
Another unrelated test was overriding `I18n.locale`, causing the TTS
test, which indirectly relies on being in English, to fail when it
randomly ended up running after the first.

### CODE REVIEW

File: dashboard/test/models/script_level_test.rb [hunk 1]
```diff
@@ -277,6 +277,7 @@ class ScriptLevelTest < ActiveSupport::TestCase
     student = create :student
     I18n.locale = 'non-default-locale'
     assert_equal script_levels[1].path, script_levels[0].next_level_or_redirect_path_for_user(student)
+    I18n.locale = I18n.default_locale
```
Comment (Bjvanminnen) [inline dashboard/test/models/script_level_test.rb]: Is there a way to do this in an after_each sort of method in ruby? Right now, if the assert on line 279 were to fail, the locale would not get reset.

**Comment by Bjvanminnen** [comment]
This test even failed in my circle run :( (but then passed on retry).

File: dashboard/test/models/script_level_test.rb (hunk 1 above)
Comment (islemaster) [reply dashboard/test/models/script_level_test.rb]: And/or we should at least do this in an `ensure` block for this one test :+1:

Comment (islemaster) [reply dashboard/test/models/script_level_test.rb]: You could also create a helper for it:

```ruby
def with_non_default_locale
  I18n.locale = 'non-default-locale'
  yield
ensure
  I18n.locale = I18n.default_locale
end

with_non_default_locale do
  assert_equal ...
end
```

Comment (Hamms) [reply dashboard/test/models/script_level_test.rb]: Good call. I added it to the global test teardown method

File: dashboard/test/test_helper.rb [hunk 2]
```diff
@@ -79,6 +79,7 @@ class ActiveSupport::TestCase
 
   teardown do
     Dashboard::Application.config.action_controller.perform_caching = false
+    I18n.locale = I18n.default_locale
```
Comment (Bjvanminnen) [inline dashboard/test/test_helper.rb:82]: Does it make more sense to reset in tear down, or in setup? (not sure of answer)

Comment (Hamms) [reply dashboard/test/test_helper.rb:82]: I picked teardown just because it's specifically undoing something that had (presumably) been done by a test.

**Pull request merged**